windows/WindowsTerminal/settings.json

```diff
@@ -7,14 +7,17 @@
         "action": "copy",
         "singleLine": false
       },
+      "id": "User.copy.644BA8F2",
```

Comment:
Looks like a typo, but looks not. This `644BA8F2` is appeared in https://github.com/microsoft/terminal/issues/17133